appshell/cefclient_win.cpp

```diff
@@ -188,7 +245,7 @@ int APIENTRY wWinMain(HINSTANCE hInstance,
   }
   else {
 	// If the shift key is not pressed, look for the index.html file 
-	if (GetAsyncKeyState(VK_SHIFT) == 0) {
+	if (!isShiftKeyDown) {
```

Comment:
This checked is moved to the top. And the check again is replaced with checking for a bit set. When launching from command line, `GetAsyncKeyState(VK_SHIFT) == 0` was returning true. The correct fix is to check for the set bit `GetAsyncKeyState(VK_SHIFT) & 0x8000`.